src/include/sof/schedule/schedule.h

```diff
@@ -99,17 +99,6 @@ struct scheduler_ops {
 	 */
 	int (*schedule_task_running)(void *data, struct task *task);
 
-	/**
-	 * Sets task into completed state along with executing additional
-	 * scheduler specific operations.
-	 * @param data Private data of selected scheduler.
-	 * @param task Task to be set into completed state.
-	 * @return 0 if succeeded, error code otherwise.
-	 *
-	 * This operation is optional.
-	 */
-	int (*schedule_task_complete)(void *data, struct task *task);
-
```

Comment:
I need it in the IPC that is currently refactored to use this DP scheduler.
Please note that `src\ipc\ipc-common.c` has `ipc_task_ops` struct that calls `ipc_complete_task` on task completion.
Please don't delete.

Reply:
ok, false alarm. Task completion is executed though I do not understand why.
Can you provide some more context @lyakh to the PR description?